src/components/picker/index.js

```diff
@@ -195,7 +195,12 @@ class Picker extends Component {
           value: nextProps.value
         };
       }
-    } else if (_.isFunction(nextProps.renderPicker) && prevState.value !== nextProps.value) {
+    } else if (
+      // Allow external change of the picker's value, only applicable to single mode
+      nextProps.mode === Picker.modes.SINGLE &&
+      _.isFunction(nextProps.renderPicker) &&
+      prevState.value !== nextProps.value
+    ) {
```

Comment:
So technically an external change of the value will not work for multi-mode pickers? 
Why? 

It feels like there's a root issue we need to fix here, no?

Reply:
I agree, however I could not think of a good example of changing the information externally in multi mode (perhaps an update from the server?), and this was the easiest fix to keep both scenarios working.
If we want a better fix for this, I suggest adding an internal ticket (maybe we want this patch until we get to that).